src/App.js

```diff
@@ -1,29 +1,28 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import LandingPage from './components/LandingPage';
+import Reservations from './components/Reservations';
+import MyReservations from './components/MyReservations';
+import DeleteReservations from './components/DeleteReservations';
+import HomePage from './components/HomePage';
+import CourseDetails from './components/CourseDetails';
+import Authentication from './components/Authentication';
+import AddCourse from './components/AddCourse';
+import DeleteCourse from './components/DeleteCourse';
 
+const App = () => (
```

Comment:
- [ ] OPTIONAL: If I review the project at 1440p resolution, the courses disappear and I only get one:

1440p:

![image](https://github.com/coderacheal/skill-habour-fe/assets/57421823/7678b6f7-74fa-45d8-95ef-4b0ef110865d)

1080p:

![image](https://github.com/coderacheal/skill-habour-fe/assets/57421823/4dc8a252-f89b-4df5-9168-8c4a22064605)

If you don't have a 144p monitor, you can reproduce the issue by zooming out:


This is just a fair warning, so you are aware of the issue.  👍

Reply:
I have checked it and on my screen it's working fine you can see:
![image](https://github.com/coderacheal/skill-habour-fe/assets/116461959/fbe956f4-13ce-49c6-83fa-fbdd2095b8b4)